### PR TITLE
[WIP] CROSSTEST

### PR DESCRIPTION
**Pull request by Makman2**
This is for https://github.com/coala-analyzer/coala/pull/1992

It tests whether the Shell commits work fine as they broke coala-bears a while back

### CODE REVIEW

File: .ci/deps.sh [hunk 1]
```diff
@@ -45,6 +45,13 @@ go get -u golang.org/x/tools/cmd/gotype
 # Ruby commands
 bundle install
 
+pip install appdirs
```
Comment (AbdealiJK) [inline .ci/deps.sh]: You need to do it after the for loop.
As the for loop runs `pyenv local $dep_version 2.7.10` which sets the python version to use for this circleci container

File: .ci/appveyor.yml [hunk 2]
```diff
@@ -39,6 +39,13 @@ install:
   - "python --version"
   - "python -c \"import struct; print(struct.calcsize('P') * 8)\""
 
+  - "%CMD_IN_ENV% pip install appdirs"
+  - git clone https://github.com/coala-analyzer/coala coala-manuall-branch-inst
+  - cd coala-manuall-branch-inst
```
Comment (AbdealiJK) [inline .ci/appveyor.yml]: May want to add `coala-manuall-branch-inst` tot he ignored folders of pytest
The tests in this folder will get run too

**Comment by AbdealiJK** [comment]
@Makman2 I've forced push to your branch.
I added the coala folder to be ignored by pytest. I think *something* was failing and changing the directory or something ... Not sure what was happening.

For commit https://github.com/coala-analyzer/coala-bears/pull/314/commits/9d30f319ccb04f6e784d50601eb34d1035933263
CircleCI seems to work - [link to build 951](https://circleci.com/gh/coala-analyzer/coala-bears/951) although your Shell commits dont work : P
Appveyor seems to work and pass ! - [link to build 1.0.784](https://ci.appveyor.com/project/coala/coala-bears/build/1.0.784/job/kr2uyejpg5w07ui5)
Not sure why travis continues to fail at `setup.py` ... [link to build 120297641](https://travis-ci.org/coala-analyzer/coala-bears/builds/120297641)

**Comment by sils1297** [comment]
@Makman2 is this branch still needed?

**Comment by Makman2** [comment]
yes please keep it

**Comment by Makman2** [comment]
closing this PR, but please don't remove the branch, crosstesting is generally useful sometimes^^

**Comment by AbdealiJK** [comment]
It;s better if you delete the branch but document it in the docs don't you
think ?

**Comment by AbdealiJK** [comment]
Created https://github.com/coala-analyzer/coala/issues/2074

**Pull request closed without merging**